### PR TITLE
Update README build instructions

### PR DESCRIPTION
**Pull request by nitrohorse**
## Description

Updates the README instructions to include `--live-reload` for serving the website locally. Helpful for not needing to manually refresh your browser on file changes.

Resolves: #none 

#### Check List 

- [x] I understand that by not opening an issue about a software/service/similar addition/removal, this pull request will be closed without merging.

- [x] I have read and understand [the contributing guidelines](https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md).

- ~~[ ] The project is [Free Libre](https://en.wikipedia.org/wiki/Free_software) and/or [Open Source](https://en.wikipedia.org/wiki/Open-source_software) Software~~

* ~~Netlify preview for the mainly edited page: ~~

* ~~Code repository of the project (if applicable):~~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,14 +14,19 @@ It's important for a website like PrivacyTools to be up-to-date. Keep an eye on
...
-1. Install [bundler](https://bundler.io/) by running `gem install bundler`.
-1. Run `bundle install` to install the required dependencies.
-1. Use `bundle exec jekyll build` to build the website. The output can be found in the `_site` directory.  Live preview is also possible by running `bundle exec jekyll serve`
-
-# Support privacytools.io
+## Developing
+
+1. Install the latest stable version of [Ruby](https://www.ruby-lang.org/en/downloads/).
+1. Install [Bundler](https://bundler.io/) (at least v2.0.1):
+	* `gem install bundler`
```
Comment (JonahAragon) [inline README.md]: It has to be 2.0.1 specifically, not at least 2.0.1: `gem install bundler:2.0.1`

File: README.md [hunk 2]
```diff
@@ -14,14 +14,19 @@ It's important for a website like PrivacyTools to be up-to-date. Keep an eye on
...
+1. Install the required dependencies:
+	* `bundle install`
+1. Build the website (the output can be found in the `_site` directory):
+	* `bundle exec jekyll build`
+1. Serve the website locally with live reloading:
+	* `bundle exec jekyll serve --incremental --livereload`
```
Comment (Mikaela) [inline README.md]: Is it `livereload` or `live-reload` actually as you use one here and one in your PR description?

File: README.md (hunk 1 above)
Comment (JonahAragon) [reply README.md]: (the reason for this, just FYI, is that 2.0.2 is the latest I think, but they are not interchangeable, and Netlify only supports 2.0.1)

File: README.md (hunk 2 above)
Comment (JonahAragon) [reply README.md]: It is indeed `livereload`

File: README.md (hunk 1 above)
Comment (nitrohorse) [reply README.md]: Updated :+1:

**Pull request merged**